src/Http/Controllers/Auth/LoginController.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Http\Controllers\Auth;
+
+use Blumilk\Meetup\Core\Http\Controllers\Controller;
+use Blumilk\Meetup\Core\Http\Requests\LoginUserRequest;
+use Blumilk\Meetup\Core\Models\User;
+use Blumilk\Meetup\Core\Services\UserLoginService;
+use Illuminate\Http\Response;
+use Illuminate\View\View;
+
+class LoginController extends Controller
+{
+    public function store()
+    {
+        return view("user.login");
+    }
+    public function login(LoginUserRequest $request): Response
+    {
+        $validated = $request->validated();
+        $user = User::where("email", $validated["email"])->first();
+        $service = new UserLoginService();
+        $service->checkUser($user, $request);
+
+        $token = $user->createToken($user->email)->plainTextToken;
+        $data = [
+            "user" => $user["email"],
+            "auth_token" => $token,
+        ];
+
+        return response()
+            ->view("user.dashboard")
+            ->header("BearerToken", $data["auth_token"]);
```

Comment:
Are we using `Bearer token`, we have normal session in app, I think. That ` "user" => $user["email"]` is needed?